internal/connect/command_offset_alter.go

```diff
@@ -0,0 +1,133 @@
+package connect
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	time2 "time"
+
+	connectv1 "github.com/confluentinc/ccloud-sdk-go-v2/connect/v1"
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/errors"
+	"github.com/confluentinc/cli/v3/pkg/examples"
+	"github.com/confluentinc/cli/v3/pkg/kafka"
+	"github.com/confluentinc/cli/v3/pkg/output"
+)
+
+func (c *offsetCommand) newAlterCommand() *cobra.Command {
+
+	cmd := &cobra.Command{
+		Use:               "alter <id>",
+		Short:             "Alter a connector's offsets'",
+		Args:              cobra.ExactArgs(1),
+		RunE:              c.alterOffset,
+		ValidArgsFunction: pcmd.NewValidArgsFunction(c.validArgs),
+		Annotations:       map[string]string{pcmd.RunRequirement: pcmd.RequireNonAPIKeyCloudLogin},
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: "Alter connector offsets for a lccId in the current or specified Kafka cluster context.",
```

Comment:
Can you replace "lccId" with something more user-friendly? I assume we could write "connect cluster" here?

Reply:
Updated